r/src/table.cpp

```diff
@@ -172,11 +172,20 @@ std::shared_ptr<arrow::Table> Table__from_dots(SEXP lst, SEXP schema_sxp) {
   std::shared_ptr<arrow::Schema> schema;
 
   if (Rf_isNull(schema_sxp)) {
-    // infer the schema from the ...
+    // infer the schema from the `...`
```

Comment:
I wonder if this whole block of code should be factored out so it can be used whether you're creating a Table or a RecordBatch. It's the same either way: return a Schema corresponding to the data given.

Reply:
Thanks. They share some code now.